pages/Product/components/Reviews/components/Header/components/AverageRating/index.jsx

```diff
@@ -0,0 +1,45 @@
+/**
+ * Copyright (c) 2017, Shopgate, Inc. All rights reserved.
+ *
+ * This source code is licensed under the Apache 2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+import RatingStars from '../../../../../../../../components/RatingStars';
+import RatingCount from '../RatingCount';
+import style from './style';
+
+/**
+ * The average rating and number of ratings for a product
+ * @param {Object} rating The rating values
+ * @returns {XML}
+ * @constructor
+ */
+const AverageRating = ({ rating }) => {
+  let average = 0;
```

Comment:
If you destructure these variables then the following block can go away.
`const { average = 0, count = 0 } = rating`;

Reply:
Done.